library/cartridge_failover_promote.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/python
+
+import pkgutil
+
+if pkgutil.find_loader('ansible.module_utils.helpers'):
+    import ansible.module_utils.helpers as helpers
+else:
+    import module_utils.helpers as helpers
+
+argument_spec = {
+    'console_sock': {'required': True, 'type': 'str'},
+    'failover_promote_params': {'required': True, 'type': 'dict'},
+}
+
+
+def failover_promote(params):
+    console_sock = params['console_sock']
+
+    specified_replicaset_leaders = params['failover_promote_params'].get('replicaset_leaders')
+    force_inconsistency = params['failover_promote_params'].get('force_inconsistency')
+
+    if not specified_replicaset_leaders:
+        return helpers.ModuleRes(changed=False)
+
+    control_console = helpers.get_control_console(console_sock)
+
```

Comment:
```suggestion
```

Reply:
I don't think it's necessary. Getting control console is separated action, this console is used below.